prow/plugins/trigger/trigger.go

```diff
@@ -34,7 +34,7 @@ import (
 
 const (
 	pluginName = "trigger"
-	lgtmLabel  = "lgtm"
+	LGTMLabel  = "lgtm"
```

Comment:
imo it's not quite so useful to make a string into a named constant if you're going to define it multiple times. filing an issue to discuss having a `labels` package for plugins label constants.

Reply:
filed https://github.com/kubernetes/test-infra/issues/9431